### PR TITLE
Updated DejaVu to 2.37

### PR DESCRIPTION
**Pull request by Roy-Orbison**
Includes diagonal block character variations. (Diagonal lines are just references to existing Unicode line drawing characters.)

image

image

### CODE REVIEW

**Comment by ZyX-I** [comment]
Can you write version in README? It is bad it is not already there. And also add information about diagonal block characters, including some reference explaining why you have chosen specific codepoints (e.g. where it is also used).

**Comment by Roy-Orbison** [comment]
Done.

**Comment by Roy-Orbison** [comment]
Not done; Github GUI commit failed.

**Comment by Roy-Orbison** [comment]
Actually worked this time.

File: DejaVuSansMono/README.rst [hunk 1]
```diff
@@ -3,4 +3,75 @@ DejaVu Sans Mono for Powerline
...
+:Patched by: `Roy-Orbison <https://github.com/Roy-Orbison>`_
+
+Includes column number, diagonal block and line characters in same positions as `Nerdfonts' <https://github.com/ryanoasis/nerd-fonts#powerline-extra-symbols>`_.
+
++-------------+-------------------+
+|**Codepoint**|**Character**      |
++-------------+-------------------+
+|::           |Branch             |
+|             |                   |
+|     E0A0    |                   |
```
Comment (ZyX-I) [inline DejaVuSansMono/README.rst]: This is way too bold for a table. One line per table entry is enough (note: one line includes “no `+---+` line”, use lighter table variant like in top README).

**Comment by Roy-Orbison** [comment]
Done. Never used reStructuredText before this. Not a fan of its ‘action at a distance’.

**Pull request merged**